_pages/en_US/installing-boot9strap-(hbl-usm).txt

```diff
@@ -38,7 +38,7 @@ In this section, you will copy the files needed to trigger the unSAFE_MODE explo
 1. Copy `boot9strap.firm` and `boot9strap.firm.sha` from the RELEASE `.zip` to the `/boot9strap/` folder on your SD card
 1. Create a folder named `3ds` on the root of your SD card if it does not already exist
     + This folder stores homebrew applications and data; it is different from the `Nintendo 3DS` folder that the console automatically generates
-1. Copy the `slotTool` folder from the RELEASE `.zip` to the `/3ds/` folder on your SD card
+1. Copy `slotTool.3dsx` folder from the `slotTool` folder inside the RELEASE `.zip` to the `/3ds/` folder on your SD card
```

Comment:
we fr repackaging everything now

though I think the XML is still needed in cases such as slotTool being ran outside of the cfg:s otherapp. Notable cases for this could be basehaxx, which can't run cfg:s otherapp for whatever reason. Yes, it isn't in the guide, but this was more so pointing at why zoogie should keep it

Reply:
(the save corrupts when you use normal jpn otherapp)